{
    "docs": "https://docs.ansible.com/ansible/2.10/dev_guide/testing/sanity/ansible-doc.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test ansible-doc` [[explain](https://docs.ansible.com/ansible/2.10/dev_guide/testing/sanity/ansible-doc.html)] failed with the error:", 
            "output": "Output on stderr from ansible-doc is considered an error.\n\nCommand \"ansible-doc -t module purestorage.fusion.fusion_api_client purestorage.fusion.fusion_array purestorage.fusion.fusion_az purestorage.fusion.fusion_hap purestorage.fusion.fusion_hw purestorage.fusion.fusion_info purestorage.fusion.fusion_nig purestorage.fusion.fusion_pg purestorage.fusion.fusion_pp purestorage.fusion.fusion_ra purestorage.fusion.fusion_sc purestorage.fusion.fusion_ss purestorage.fusion.fusion_tenant purestorage.fusion.fusion_tn purestorage.fusion.fusion_ts purestorage.fusion.fusion_volume\" returned exit status 0.\n>>> Standard Error\n/usr/local/lib/python3.8/dist-packages/pkg_resources/__init__.py:122: PkgResourcesDeprecationWarning: 0.23ubuntu1 is an invalid version and will not be supported in a future release\n  warnings.warn(\n/usr/local/lib/python3.8/dist-packages/pkg_resources/__init__.py:122: PkgResourcesDeprecationWarning: 0.1.36ubuntu1 is an invalid version and will not be supported in a future release\n  warnings.warn(\n[WARNING]: Collection purestorage.fusion does not support Ansible version\n2.10.5"
        }
    ], 
    "verified": false
}
